src/components/Search/engines/StopFinder.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
 
+import { StopsAPI } from 'mobility-toolbox-js/api';
 import Engine from './Engine';
 
 class StopFinder extends Engine {
   constructor(endpoint = 'https://api.geops.io/stops/v1/', options = {}) {
     super();
     this.options = options;
-    this.endpoint = endpoint;
+    this.api = new StopsAPI({
+      url: endpoint,
+      apiKey: options.apiKey,
+    });
   }
 
   search(value) {
-    const optionsString = `${Object.keys(this.options)
-      .map((p) => `${p}=${this.options[p]}`)
-      .join('&')}`;
-    return fetch(
-      `${this.endpoint}?&q=${encodeURIComponent(value)}&key=${
-        this.apiKey
-      }&${optionsString}`,
-    )
-      .then((data) => data.json())
-      .then((featureCollection) => featureCollection.features);
+    return this.api.search({
+      q: value,
+    });
```

Comment:
What happens with the use of options and the encodeURIComponnet function ? If they were there it was for a resaon I guess

Reply:
Indeed forgot that! Thanks

Reply:
If that is it, we could merge it.

Reply:
and options ?

Reply:
Of course you are right, options are added as well